.github/workflows/OpenAPIInSync.yml

```diff
@@ -1,29 +0,0 @@
-name: openapi.json in sync (prod)
```

Comment:
Do we want to remove this file? I am not 100% sure why this is, but wanted to verify if it is not used.

Reply:
@mkholjuraev, It is listed to be executed only on 
      - prod
      - prod-stable
      - prod-beta
branches. I haven't dug into the job much, but since we never deploy to other branches and thus the script hasn't been executed for a long time, I see no value in this file.

Reply:
Ah, yes makes sense